packages/react-vapor/src/components/actions/tests/SecondaryActions.spec.tsx

```diff
@@ -88,7 +92,11 @@ describe('Actions', () => {
             />
         );
 
-        expect(wrapper.find('div.dropdown').props().style.cursor).toBe('default');
+        expect(wrapper.find('div.dropdown').props()).toMatchObject({

```

Comment:
When the linter passes, it changed the test by the `toHaveStyle` method, which gave this `expect (wrapper.find('div.dropdown').props()).toHaveStyle({cursor: 'default'});`. But the linter was wrong because we are not testing `style` but `props` :smile: